sources/PTYSession.h

```diff
@@ -728,6 +728,8 @@ backgroundColor:(NSColor *)backgroundColor;
 // Call refresh on the textview and schedule a timer if anything is blinking.
 - (void)refresh;
 
+- (void)textViewSelectionDidChangeToTruncatedString:(NSString *)selection;
```

Comment:
Delete this line. It's sufficient to declare it in the delegate protocol.

Reply:
Weird; I thought I'd already removed that.   Gone for sure now....